apps/storybook/src/stories/datacom-list.stories.tsx

```diff
@@ -135,41 +126,37 @@ export const UnorderedListWithNestedOrderedItems = () => {
 };
 
 export const StandaloneOrderedList = (args) => {
+  const { heading } = args;
   return (
     <div>
       <DatacomList variant="ordered" {...args}>
-        <DatacomLi {...args} heading={true}>
-          List content item
+        <DatacomLi {...args} heading={heading} style={{ marginBottom: '12px' }}>
+          Lorem ipsum dolor sit amet
         </DatacomLi>
-        <P>Lorem ipsum dolor sit amet</P>
-        <DatacomLi {...args} heading={true}>
-          List content item
+        <DatacomLi {...args} heading={heading} style={{ marginBottom: '12px' }}>
+          Lorem ipsum dolor sit amet
         </DatacomLi>
-        <P>Lorem ipsum dolor sit amet</P>
-        <DatacomLi {...args} heading={true}>
-          List content item
+        <DatacomLi {...args} heading={heading}>
+          Lorem ipsum dolor sit amet
         </DatacomLi>
-        <P>Lorem ipsum dolor sit amet</P>
       </DatacomList>
     </div>
   );
 };
 export const StandaloneUnorderedList = (args) => {
+  const { heading } = args;
   return (
     <div>
       <DatacomList {...args} variant="Unordered">
-        <DatacomLi {...args} heading={true}>
-          List content item
+        <DatacomLi {...args} heading={heading} style={{ marginBottom: '12px' }}>
```

Comment:
Considering this margin-bottom is only needed when you have a heading on the item, perhaps you can move this margin to be applied on the "dc-li-marker" class you've added?

Reply:
Thanks, I will do add it.